packages/extractor/package.json

```diff
@@ -57,13 +68,11 @@
 		"dev:doc": "typedoc --watch",
 		"dev:test": "pnpm vitest watch --passWithNoTests --ui --open=false --workspace \"../../vitest.workspace.ts\" --project \"@svelte-docgen/extractor\" ",
 		"fix": "pnpm run \"/^fix:.*/\" ",
-		"fix:biome": "biome check . --verbose --write",
```

Comment:
I think we could leave this script, and change to [`biome lint`](https://biomejs.dev/reference/cli/#biome-lint) instead, as I believe it doesn't run formatter.

Unless we plan to ditch Biome for ESLint too _(because that's what used on the official Svelte repository - to align with their coding standards & CI setup)_.

Reply:
Reverted the `fix:biome` and `lint:biome` commands. I think it’s fine to keep using `check` instead of `lint`, since the formatter function is disabled in `biome.json`.

Reply:
For now, I’ll keep the linter as Biome (I actually quite like it) in this PR, but we might consider switching to ESLint eventually.